code/modules/paperwork/faxmachine.dm

```diff
@@ -226,7 +244,7 @@ GLOBAL_LIST_EMPTY(fax_blacklist)
 			usr.drop_item()
 			card.forceMove(src)
 			scan = card
-	SSnanoui.update_uis(src)
+		SStgui.update_uis(src)
```

Comment:
You accidentally double-indented this, when its meant to be single-indented (See old diff and spacing)